### PR TITLE
Fixes building issues with 4.24

### PR DESCRIPTION
**Pull request by alexandrosk0**
Fixes building issues with 4.24.

### CODE REVIEW

File: Source/DedicatedServer/Private/DedicatedServer.cpp [hunk 1]
```diff
@@ -86,7 +86,7 @@ void FDedicatedServerModule::StartupModule()
 											// ToDo: find a better way which also respects the message field...
 											FString sRequiredVersion = FString::FromInt( pResponse->GetIntegerField( TEXT( "required_version" ) ) );
 											UE_LOG( LogDedicatedServer, Warning, TEXT( "The server is outdated( %s -> %s ), restarting..." ), *sVersion, ( sRequiredVersion.Equals( sVersion ) ? *sRequiredVersion : *Str2SemVer( sRequiredVersion ) )/*, pResponse->GetStringField( "message" )*/ );
-											GIsRequestingExit = true;
+											RequestEngineExit(TEXT("The server is outdated"));
```
Comment (neico) [inline Source/DedicatedServer/Private/DedicatedServer.cpp]:
```suggestion
											RequestEngineExit( TEXT( "The server is outdated" ) );
```

File: Source/DedicatedServer/Private/ServerConsoleWindows.cpp [hunk 2]
```diff
@@ -284,8 +284,8 @@ void DumpConsoleHelp();
 
 	COORD FServerConsole::GetCursorPosition()
 	{
-		COORD hCursorPosition = 0;
-
+		COORD hCursorPosition;
+		hCursorPosition.X = hCursorPosition.Y = 0;
```
Comment (neico) [inline Source/DedicatedServer/Private/ServerConsoleWindows.cpp]:
```suggestion
		COORD hCursorPosition = {};
```

**Review by neico — CHANGES_REQUESTED** [review]
Mostly style related changes, the rest should be ok at least.

**Comment by alexandrosk0** [comment]
My bad for adding the autocompletion in this pull request.

**Review by neico — APPROVED** [review]

**Pull request merged**